xml/storage_multipath.xml

```diff
@@ -1807,7 +1733,8 @@ blacklist_exceptions {
     Starting with &productname; 12 SP2, the multipath tools support the option
     <literal>find_multipaths</literal> in the <literal>defaults</literal>
     section of <filename>/etc/multipath.conf</filename>. Simply put, this
-    option prevents multipath and <systemitem
+    option prevents multipath and
```

Comment:
maybe getting a little picky here, but 'simply put' is unnecessary

Reply:
OK. Fixed.